src/pharmpy/modeling/blq.py

```diff
@@ -65,6 +104,17 @@ def transform_blq(model: Model, method: str = 'm4', lloq: Optional[float] = None
 
     Examples
     --------
+    >>> from pharmpy.modeling import *
+    >>> model = load_example_model("pheno")
+    >>> model = transform_blq(model, method='m3', lloq=0.1)
+    >>> model.statements.find_assignment("Y")
+        ⎧  EPS₁⋅W + F    for DV ≥ LLOQ
+        ⎪
+        ⎨   ⎛-F + LLOQ⎞
+        ⎪PHI⎜─────────⎟    otherwise
+    Y = ⎩   ⎝    SD   ⎠
+
+
```

Comment:
Is this example necessary? Was thinking since it's so similar (in terms of how you call it etc)

Reply:
I was thinking more as an illustration of what is changed in the model rather than showing how the function is used.

Reply:
Okay didn't fully finish my sentence, sorry. I meant that it is so similar to the next example, it's only a part of the Y-statement that changed.